dist/grapes.js

```diff
@@ -31827,10 +31885,14 @@ module.exports = _backbone2.default.View.extend({
    * @private
    */
   getOptions: function getOptions() {
+    var _this = this;
+
     var result = '';
     this.collection.each(function (device) {
-      var name = device.get('name');
-      result += '<option value="' + name + '">' + name + '</option>';
+      var id = device.id,
+          className = device.className;
+
+      result += '\n        <div class="' + _this.ppfx + 'radio-item">\n          <input id="' + id + '" class="' + _this.ppfx + 'radio" \n          type="radio" \n          name="device" \n          value="' + device.get('name') + '" \n          ' + (device.get('active') ? ' checked' : '') + '>\n          <label class="' + _this.ppfx + 'radio-item-label ' + className + '" \n          for="' + id + '"\n          title="' + device.get('name') + '"></label>\n        </div>\n      ';
```

Comment:
Since I don't have a ton of context I could certainly be wrong, but If this is html I don't think all the newlines and spaces are relevant.